scripts/digest_inputs

```diff
@@ -93,18 +100,20 @@ add_vars ZIP_LOCATION ZIP_NAME ZIP_NAME_GUESSED
 TARBALL_ROOT="${REPO_NAME}_${TARGET}_${OS}_${ARCH}_${VERSION}_${REVISION}"
 DEFAULT_TARBALL_NAME="${TARBALL_ROOT}.docker.tar"
 DEFAULT_DEV_TARBALL_NAME="${TARBALL_ROOT}.docker.dev.tar"
+DEFAULT_REDHAT_TARBALL_NAME="${TARBALL_ROOT}.docker.redhat.tar"
 
-# TARBALL_NAME and DEV_TARBALL_NAME may be set explicitly by tests, by
+# (DEV_|REDHAT_)?TARBALL_NAME may be set explicitly by tests, by
```

Comment:
Should `(DEV_|REDHAT_)?TARBALL_NAME` be `DEFAULT_DEV_REDHAT_TARBALL_NAME` here?

Reply:
This is a shorthand, using a regular expression to name three  different variables: `TARBALL_NAME` `DEV_TARBALL_NAME` and `REDHAT_TARBALL_NAME`. I've seen this used somewhere in the Go stdlib documentation, but maybe it's not so clear?